src/camp/dto/camp.entity.ts

```diff
@@ -0,0 +1,29 @@
+import { Camp } from '@prisma/client';
+
+export class CampEntity {
+  id: string;
+  createdAt: Date;
+  updatedAt: Date;
+  year: string;
+  name: string;
+  url: string | null;
+  contact_email: string | null;
+  hometown: string | null;
+  description: string | null;
+  location_string: string | null;
```

Comment:
Same deal with the location string here. Once it's moved into the location object this'll need to be updated